### ISSUE TITLE
CodePath Review

### ISSUE THREAD

**Issue by megazerohex** [megazerohex/My-Twitter#2]
Please review my app (Twitter part 2), thanks. /cc @codepathreview

**Comment by tuhoag** [comment]
It does not appear that you have added a README file for Assignment 5 to your Repo. Please add a README file, and use the README template to fill out the README. Be sure to also include an animated gif in your README to show a demo of your finished app. When properly linked, the gif should render (animate) within your README file. 

It looks like you still need to complete some required user stories for your submission to be complete. 

Your assignment needs to be updated within 24hrs in order for you to receive credit for this assignment.

**Comment by megazerohex** [comment]
Yes it is there, but in the secondWeek branch. Look https://github.com/megazerohex/My-Twitter/tree/second-version
Thanks, Jamel Peralta Coss

Date: Tue, 23 Feb 2016 03:24:47 -0800

**Comment by tuhoag** [comment]
:+1: Nice work! The point of this homework was to explore an example of a full MVC application with a RESTful API. 

We have a detailed [Project 4 Feedback Guide](http://courses.codepath.com/snippets/ios_university/feedback_guides/project_4_feedback.md) which covers the best practices for implementing this assignment. Read through the feedback guide point-by-point to determine ways you might be able to improve your submission. You should consider going back and implementing these improvements as well. Keep in mind that one of the most important parts of iOS development is learning the correct patterns and conventions.

If you have any particular questions about the assignment or the feedback, feel free to reply here or email us at universitysupport@codepath.com.
